v1.1.1/js/Toolbar.js

```diff
@@ -12,7 +12,10 @@ function Toolbar(loopy){
 	var buttons = [];
 	var buttonsByID = {};
 	self.dom = document.getElementById("toolbar");
-	self.addButton = function(options){
+	self.templatesBar = document.createElement("div"); // What even is a "dom"?
```

Comment:
DOM stands for Document Object Model and is more or less what allows us to make changes to the HTML via the Javascript if I understand correctly.
Further reading:
https://www.w3schools.com/js/js_htmldom.asp